samples/unit-tests/series-pie/label-align/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test("Change of label alignment after add", function (assert) {
+QUnit.test("Change of label alignment after add (#4652)", function (assert) {
```

Comment:
This test fails because in the karma template, the `container` is offset by 8px. Work around that by adding the `offsetLeft` to the expected result: 

```js
    assert.close(
        g.getBoundingClientRect().left,
        100 + document.getElementById('container').offsetLeft,
        1, // +/- 0.5px in Edge
        "Box is left aligned"
    );
```